site/cypress/e2e/Cypress-Feature/forums.spec.js

```diff
@@ -10,7 +10,6 @@ const reply2 = 'Cypress Reply 2 Cypress';
 const reply3 = 'Cypress Reply 3 Cypress';
 const merged1 = 'Merged Thread Title: '.concat(title3, '\n\n', content3);
 const merged2 = 'Merged Thread Title: '.concat(title2, '\n\n', content2);
-const attachment1 = 'sea_animals.png';
```

Comment:
I think leaving this as a variable is better, one because it shouldn't add any flakeyness to the test, which is what this PR is attempting to fix, and two, if we decide to add more attachments later, to keep this file consistent they should be added as variables to the top.

Reply:
I removed it because there were existing lines of code which didn't use the variable. I'll change those too then.